runtimes/pythonrt/pythonrt.go

```diff
@@ -642,117 +642,44 @@ func (py *pySvc) Call(ctx context.Context, v sdktypes.Value, args []sdktypes.Val
 	return sdktypes.NewBytesValue(rm.Value), nil
 }
 
-// TODO: This is long and dog ugly, find a better way.
-func (py *pySvc) handleCall(ctx context.Context, msg CallMessage) error {
-	var value any
-
-	switch msg.FuncName {
-	case "sleep":
-		if len(msg.Args) != 1 {
-			return fmt.Errorf("sleep: expected 1 argument, got %d", len(msg.Args))
-		}
-
-		sleep, ok := msg.Args[0].(float64)
-		if !ok {
-			return fmt.Errorf("sleep: expected float64, got %T", msg.Args[0])
-		}
-
-		d := time.Duration(sleep*1000) * time.Millisecond
-		args := []sdktypes.Value{
-			sdktypes.NewStringValue("sleep"),
-			sdktypes.NewDurationValue(d),
-		}
-
-		_, err := py.cbs.Call(ctx, py.xid.ToRunID(), py.syscallFn, args, nil)
-		if err != nil {
-			py.log.Error("call sleep", zap.Error(err))
-			return err
-		}
-		value = nil
-	case "subscribe":
-		if len(msg.Args) < 1 || len(msg.Args) > 2 {
-			return fmt.Errorf("subscribe: expected 1 or 2 arguments, got %d", len(msg.Args))
-		}
-
-		connection, ok := msg.Args[0].(string)
-		if !ok {
-			return fmt.Errorf("subscribe: expected string ID, got %T", msg.Args[0])
-		}
-
-		args := []sdktypes.Value{
-			sdktypes.NewStringValue("subscribe"),
-			sdktypes.NewStringValue(connection),
-		}
-
-		filter := sdktypes.NewStringValue("true")
-		if len(msg.Args) == 2 {
-			v, ok := msg.Args[1].(string)
-			if !ok {
-				return fmt.Errorf("subscribe: expected string filter, got %T", msg.Args[1])
-			}
-			filter = sdktypes.NewStringValue(v)
-		}
-		args = append(args, filter)
-
-		id, err := py.cbs.Call(ctx, py.xid.ToRunID(), py.syscallFn, args, nil)
+func packSyscallArgs(msg CallMessage) ([]sdktypes.Value, map[string]sdktypes.Value, error) {
+	args := make([]sdktypes.Value, len(msg.Args)+1)
+	args[0] = sdktypes.NewStringValue(msg.FuncName)
+	// Can't use kittehs.Transform since we want to handle errors.
```

Comment:
there's TransformError

Reply:
> there's TransformError

Thanks. However `args[0]` is something different and then I'll need another slice and copy it back. Will keep as-is.